monitor/utils/events.js

```diff
@@ -95,11 +97,52 @@ async function main(mode) {
   })).map(normalizeEvent)
 
   logger.debug("calling foreignBridge.getPastEvents('UserRequestForAffirmation')")
-  let foreignToHomeRequests = (await getPastEvents(foreignBridge, {
+  foreignToHomeRequests = (await getPastEvents(foreignBridge, {
     event: v1Bridge ? 'Withdraw' : 'UserRequestForAffirmation',
     fromBlock: MONITOR_FOREIGN_START_BLOCK,
     toBlock: foreignBlockNumber
   })).map(normalizeEvent)
+
+  // append old AMB UserRequestForSignature and UserRequestForAffirmation events
+  if (bridgeMode === BRIDGE_MODES.ARBITRARY_MESSAGE) {
+    // replace UserRequestForSignature abi with the old one
+    const OLD_HOME_ABI = HOME_ABI.filter(x => x.name !== 'UserRequestForSignature')
```

Comment:
It is more straightforward way is to add a json with this ABI in `commons/abis.js` rather than perform replacement evert time the monitor run. For example, we already have `ERC20_BYTES32_ABI` there as well as `HOME_V1_ABI` and `FOREIGN_V1_ABI` loaded from `v1Abis.js`.